content/200-concepts/100-components/01-prisma-schema/09-index-configuration.mdx

```diff
@@ -129,17 +142,58 @@ model Post {
+
+The following example demonstrates adding a `@@fulltext` index to the `title` and `content` fields of a `Post` model:
+
+```prisma file=schema.prisma
+model Post {
+  id      Int    @id
+  title   String @db.VarChar(255)
+  content String @db.Text
+
+  @@fulltext([title, content])
+}
+```
+
+On MongoDB, the `fullTextIndex` preview feature can be combined with the `extendedIndexes` preview feature to add ascending or descending columns to your full-text index:
+
+```prisma file=schema.prisma
+generator js {
+  provider        = "prisma-client-js"
+  previewFeatures = ["fullTextIndex", "extendedIndexes", "mongoDb"]
+}
+
+model Post {
+  id      String @id @map("_id") @db.ObjectId
+  title   String @db.VarChar(255)
+  content String @db.Text
+
+  @@fulltext([title(sort: Desc), content])
+}
+```
+
 ## Upgrading from previous versions
 
 <Admonition type="warning">
 
-This is a **breaking change** when updating from certain existing databases. After updating, always run `prisma db pull` to introspect the existing database before migrating.
+These are **breaking changes** when updating from certain existing databases. After updating, always run `prisma db pull` to introspect the existing database before migrating.
 
 </Admonition>
 
+### Upgrading with existing `sort` and `length` constraints and indexes
+
 A breaking change can occur in the following situations:
 
 - **Existing sort constraints and indexes:** earlier versions of Prisma will assume that the desired sort order is _ascending_ if no order is specified explicitly. This means that this is a breaking change if you have existing constraints or indexes that are using descending sort order and migrate your database without first specifying this in your data model.
 - **Existing length constraints and indexes:** in earlier versions of Prisma, indexes and constaints that were length constrained in MySQL could not be represented in the Prisma schema. Therefore `prisma db pull` was not fetching these and you could not manually specify them. When you ran `prisma db push` or `prisma migrate dev` they were ignored if already present in your database. Since you are now able to specify these, migrate commands will now drop them if they are missing from your data model but present in the database.
 
 In both of the cases above unwanted changes to your database can be prevented by properly specifying these properties in your data model where necessary. **The easiest way to do this is to use `prisma db pull` to retrieve the sort and length arguments where applicable.** Alternatively, you could also add these arguments manually. This should be done before using `prisma db push` or `prisma migrate dev`.
+
+### Upgrading with existing fulltext indexes
+
+Earlier versions of Prisma converted fulltext indexes using the `@@index` attribute rather than the `@@fulltext` attribute. After enabling the `fullTextIndex` preview feature, run `prisma db pull` to convert these indexes to `@fulltext` before migrating.
```

Comment:
```suggestion
Earlier versions of Prisma converted fulltext indexes using the `@@index` attribute rather than the `@@fulltext` attribute. After enabling the `fullTextIndex` preview feature, run `prisma db pull` to convert these indexes to `@@fulltext` before migrating.
```

Reply:
Thanks, fixed as part of rearrange